docs/rst/use_cases/ros_cloud.rst

```diff
@@ -0,0 +1,144 @@
+are required in order to direct dataflow to each of the pods. A LoadBalancer will forward messages reaching the cluster
+to the WAN participant of the cloud router, and a ClusterIP service will be in charge of delivering messages from the
+local DS to the listener pod. Following are the settings needed to launch these services in |k8s|:
+
+.. literalinclude:: ../../resources/use_cases/ros_cloud/load-balancer-service.yaml
+    :language: yaml
+
+.. literalinclude:: ../../resources/use_cases/ros_cloud/local-service.yaml
+    :language: yaml
+
+.. note::
+
+    An `Ingress <https://kubernetes.io/docs/concepts/services-networking/ingress/>`_ needs to be configured for the
+    LoadBalancer service to make it externally-reachable. In this example we consider the assigned public IP address to
+    be ``2.2.2.2``.
+
+The configuration file used for the cloud router will be provided by setting up a `ConfigMap <https://kubernetes.io/docs/concepts/configuration/configmap/>`_:
+
+.. literalinclude:: ../../resources/use_cases/ros_cloud/ConfigMap.yaml
+    :language: yaml
+
+Following is represented the overall configuration of our |k8s| cluster:
+
+.. figure:: /rst/figures/vulcanexus_k8s.png
+
+
+DDS-Router deployment
+---------------------
+The cloud router is launched from within a Docker image, which uses as configuration file the one hosted in
+the previously set up ConfigMap. This Docker image needs to be built and made available to the |k8s| cluster for using
+|ddsrouter|, which can be accomplished by providing the following
+:download:`Dockerfile <../../resources/use_cases/ros_cloud/Dockerfile_ddsrouter>`. Assuming the name of the generated
+Docker image is ``ddsrouter:main``, the cloud router will then be deployed with the following settings:
+
+.. literalinclude:: ../../resources/use_cases/ros_cloud/ddsrouter.yaml
+    :language: yaml
+
+
+Listener deployment
+-------------------
+A suitable Docker image must also be provided in the context of the cluster in order to use *ROS 2*. We will use
+``ros:galactic`` as basis for this image, install ``demo-nodes-cpp``, and include a parser that will allow us to specify
+the port and IP address of the local DS. This can be achieved by using the following Dockerfile and entrypoint:
+
+.. literalinclude:: ../../resources/use_cases/ros_cloud/Dockerfile_listener
+    :language: Dockerfile
+
+.. literalinclude:: ../../resources/use_cases/ros_cloud/run.bash
+    :language: bash
+
+Now, assuming the name of the built image is ``ros2-demo-nodes:galactic``, the listener pod can be deployed by providing
+the following configuration:
+
+.. literalinclude:: ../../resources/use_cases/ros_cloud/listener.yaml
+    :language: yaml
+
+
+Once all these components are up and running, communication should have been established between talker and listener
+nodes. Feel free to interchange the locations of the ROS nodes by slightly modifying the provided configuration files,
```

Comment:
```suggestion
nodes. (Maybe) Depending on the k8s platform you are using, you may have to refresh in order to see the logs of the ``stdout`` of your ``listener``.

Feel free to interchange the locations of the ROS nodes by slightly modifying the provided configuration files,
```
I would separate both sentences, as they are not related. This way, the last sentence is more visible.

Reply:
:+1: